### ISSUE TITLE
Error! (file) is not UTF-8 encoded

### ISSUE THREAD

**Issue by Nizami91** [jupyter/notebook#5003]
I cannot open the saved .npy file. As I click on the file to open it, I get the following text:

Error! C:\Users\Ozgun\workspace\saved_test.npy is not UTF-8 encoded
Saving disabled.
See Console for more details.

Please help.

**Comment by jishanshaikh4** [comment]
It seems like this problem already has a solution [here](https://github.com/mtobeiyf/audio-classification/issues/2) at an audio-classification repository.

**Comment by mzr4332** [comment]
I was facing a similar problem because I was trying to open a compressed(zipped) folder that had an extension '.ipynb'. So I moved the actual IPYNB File in it to 'Desktop' and was then able to access it using jupyter.

**Comment by majorfisg** [comment]
> Я столкнулся с аналогичной проблемой, потому что я пытался открыть сжатую(сжатую) папку, которая имела расширение '.ипынб'. Поэтому я переместил фактический файл IPYNB в нем на "рабочий стол", а затем смог получить доступ к нему с помощью jupyter.

The exact same problem. The .gz format archive broke the .ipynb file. Do not write in detail what they did?

**Comment by onassis28** [comment]
use encoding 'cp1252'

**Comment by awais-web** [comment]
Just double check the file format you are trying to open. is it 'ipynb'?

**Comment by scawa1952** [comment]
> It seems like this problem already has a solution [here](https://github.com/mtobeiyf/audio-classification/issues/2) at an audio-classification repository.

No it isn't.  This is a circular reference that refers me back to this thread.  Sounds like this is a Stack Overflow forum.

**Comment by xiaodong-Ren** [comment]
I had the same problem,have you solved it?I'm on the verge of collapse.

**Comment by kevin-bates** [comment]
Hi @xiaodong-Ren - If you could provide traceback and/or contextual information regarding, others may be able to assist.  Thanks.

**Comment by RokayaNeeraj** [comment]
f = open("CORONA.csv","w",encoding='utf-8'). Use it behind and it will resolve the problem

**Comment by codexsys-7** [comment]
> 

Yes, the jupyter notebook cant read that file, you need to open it where you have saved it.
Thanks, Vallabh1822, saved my day.

**Comment by FardinN77** [comment]
Hi to all,
I have the same problem when I want to load the MNIST dataset with this format **(mnist. npz)** in Jupyter; I use from below codes to import the file but appeared the error that showed my file does not exist (FileNotFoundError: [Errno 2] No such file or directory: 'MNIST_data/mnist.npz'). When I checked it from my files and tried to open it, another error appeared that said mnist.npz is not UTF-8 encoded.

How can I fix it? Help me pls.

My codes :
import numpy as np
path = "MNIST_data/mnist.npz"
with np.load(path, allow_pickle=True) as f:
    print(type(f))
    x_train, y_train = f['x_train'], f['y_train']
    x_test, y_test = f['x_test'], f['y_test']
